apps/tlon-web/src/state/unreads.ts

```diff
@@ -48,7 +48,7 @@ export interface UnreadsStore {
   update: (unreads: Activity) => void;
 }
 
-export const unreadStoreLogger = createDevLogger('UnreadsStore', false);
+export const unreadStoreLogger = createDevLogger('UnreadsStore', true);
```

Comment:
```suggestion
export const unreadStoreLogger = createDevLogger('UnreadsStore', false);
```

Do we want this left on?